aws-athena/views/model-vw_card_res_input.sql

```diff
@@ -10,9 +10,19 @@ filled with the following steps:
    PIN (ONLY for things that don't vary at the property level, such as census
    tract statistics. Property characteristics are NOT filled)
 
-WARNING: This is a very heavy view. Don't use it for anything other than making
-extracts for modeling
+This view is "materialized" (made into a table) daily in order to improve
+query performance and reduce data queried by Athena. The materialization
+is triggered by sqoop-bot (runs after Sqoop grabs iasWorld data)
 */
+{{
+    config(
+        materialized='table',
+        partitioned_by=['year'],
+        bucketed_by=['meta_pin'],
+        bucket_count=1
+    )
+}}
```

Comment:
**[Thought, non-blocking]** Now that these views are being materialized, I wonder if it's misleading that they have the `vw_*` prefix and live in `aws-athena/views` instead of `aws-athena/ctas`? The tradeoff of moving/renaming them is we would probably have to tweak a bunch of downstream consumers like the model pipelines. I could go either way, just curious if you have thoughts about long-term organization.

Reply:
Let's keep it as-is for the moment. We can always refactor these later since there's only the two of them.